content/installation/migrating-manager.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: bt_wiki
+title: Reinstalling and Migrating Cloudify Manager
+category: Installation
+draft: true
+weight: 400
+
+---
+
+To move your Cloudify Manager to another host, you must use same process as a reinstall. One reason to migrate is to move your Cloudify Manager to a host with more resources. You can do this in one of these ways:
```

Comment:
This file just seems to be a summarised rewrite of the actual snapshot restore procedure. It might be better if it just stated that migrating a manager is done by following the snapshots procedure, and linked to it? Otherwise there's a high chance that the two different lists of steps will become out of sync and cause confusion.